app/src/main/java/com/infomaniak/mail/data/api/MailApi.kt

```diff
@@ -50,10 +50,10 @@ object MailApi {
         return ApiRepository.getThreads(mailboxUuid, folder.id).data?.threads?.map { it.initLocalValues() }
     }
 
-    fun fetchMessages(thread: Thread): List<Message> {
+    fun fetchMessages(thread: Thread): List<Pair<Message, Boolean>> {
         return thread.messages.map { realmMessage ->
             if (realmMessage.fullyDownloaded) {
-                realmMessage
+                realmMessage to true
```

Comment:
This realm object is not a managed realm ?

Reply:
Oh yes it is! I don't need this boolean, thanks.